packages/algorithm/package.json

```diff
@@ -26,9 +26,9 @@
     "api": "api-extractor run --local --verbose",
     "build": "npm run build:src && rollup -c",
     "build:src": "tsc --build",
-    "build:test": "tsc --build tests && cd tests && webpack",
+    "build:test": "npm run clean:test && tsc --build tests && cd tests && rollup -c",
```

Comment:
I wasn't sure if I should add `clean:test` here, but without it the `build:test` script/command is not idempotent because it creates a `tsconfig.tsbuildinfo` file, which I found prevents a second run of the `build:test` command from picking up intervening changes in the test code.